tests/utils/test_utility.py

```diff
@@ -70,3 +105,5 @@ def test_different_cache(linear_dataset, memcache_client_config):
     )
 
     assert u1.signature != u2.signature
+    assert u1.signature == u1.signature
+    assert u2.signature == u2.signature
```

Comment:
Is this correct?

Reply:
Yes. I just wanted to be extra sure since it's a property.
Right now it's redundant though.